src/main/java/org/jenkinsci/plugins/github/util/FluentIterableWrapper.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
```

Comment:
Usage of Apache code in the MIT plugin requires additional notes on the top level.
Was it possible not to copy the code?